Cargo.lock

```diff
@@ -5401,7 +5400,7 @@ version = "1.6.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
Hm, why this changed?

Reply:
The version specified by twox-hash is ">=0.1, <2". This means both 0.1.10 and 1.0.0 work. Which one is actually used seems to be a matter of luck.